modules/querying-ingested-content-in-a-llama-model.adoc

```diff
@@ -66,19 +66,72 @@ models = client.models.list()
  Model(identifier='ibm-granite/granite-embedding-125m-english', metadata={'embedding_dimension': 768.0}, api_model_type='embedding', provider_id='sentence-transformers', provider_resource_id='ibm-granite/granite-embedding-125m-english', type='model', model_type='embedding')]
 ----
 
-. In a new notebook cell, select the first LLM in your list of registered models:
+. Select the first LLM:
 +
 [source,python]
 ----
 model_id = next(m.identifier for m in models if m.model_type == "llm")
 ----
 
-. In a new notebook cell, define the `vector_db_id`, which is a unique name that identifies your vector database, for example, `my_milvus_db`. If you do not know your vector database ID, contact an administrator.
+. (Optional) If you have *not* already created a vector store, select an embedding model (for registration in the next step):
```

Comment:
Suggest removing these parenthetical, as they seem necessary to a user who chooses to take this optional step.